src/components/BorrowerProfile/LendersAndTeams.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+	<section>
+		<!-- Display Content  v-if="!loading && this.totalItemCount" -->
+		<div>
+			<h2 class="tw-text-h2">
+				{{ sectionTitle }}
+			</h2>
+
+			<div class="tw-mb-3">
+				<kv-material-icon
+					class="tw-h-2.5 tw-pointer-events-none tw-inline-block tw-align-middle"
+					:icon="mdiLightningBolt"
+				/>
+
+				<kv-loading-placeholder
+					v-if="loading"
+					:style="{width: 20 + (Math.random() * 15) + '%', height: '14px'}"
+					class="tw-ml-1 tw-inline-block tw-align-middle"
+				/>
+				<span v-else>
+					{{ poweredByText }}
+				</span>
+			</div>
+
+			<div v-if="loading" class="tw-grid tw-grid-cols-2 tw-gap-2">
+				<!-- Loading placeholder for team or lender item elements -->
+				<div v-for="i in 5" :key="i" class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-mb-3">
+					<kv-loading-placeholder class="tw-rounded md:tw-hidden tw-mb-1.5" :style="{height: '128px'}" />
+					<kv-loading-placeholder
+						class="tw-rounded md:tw-block tw-hidden tw-mr-2" :style="{width: '88px', height: '88px'}"
```

Comment:
Adding `flex-shrink-0` will prevent these squares from getting squished
![image](https://user-images.githubusercontent.com/187937/125529450-01a95485-5deb-4398-b64c-2446f8a8a5f7.png)

Reply:
Indeed, these are just placeholders until we drop in the real component.